sdk/core/Azure.Core/tests/common/Azure.Core.Tests.Common.csproj

```diff
@@ -7,6 +7,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\..\..\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
+    <ProjectReference Include="..\..\..\System.ClientModel\src\System.ClientModel.csproj" />
```

Comment:
Why is this needed?

Reply:
To test the unbranded path for new model serialization which is the first implementation to be shipped after this goes live